components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/Oauth2ScopeConstants.java

```diff
@@ -0,0 +1,84 @@
+/*
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.wso2.carbon.identity.oauth2;
+
+public class Oauth2ScopeConstants {
+
+    public static final String TENANT_NAME_FROM_CONTEXT = "TenantNameFromContext";
+
+    public static final int MAX_FILTER_COUNT = 30;
+
+    public enum ErrorMessages {
+        ERROR_CODE_FAILED_TO_REGISTER_SCOPE("18001", "Error occurred while registering scopes. "),
```

Comment:
start the error codes from "41xxx" for client exceptions adn server exceptions from "51xxx"